labs/architecture-setup/hello-flask/requirements.txt

```
@@ -1,3 +1,10 @@
 -i http://nexus-infra.apps.ocp4.example.com/repository/pypi/simple --trusted-host nexus-infra.apps.ocp4.example.com
 Flask==2.1.0
 gunicorn==20.1.0
+click==8.1.7
+importlib-metadata==6.8.0
+itsdangerous==2.1.2
+Jinja2==3.1.2
+MarkupSafe==2.1.3
+Werkzeug==3.0.0
```

Version 3 is the problematic one
```suggestion
Werkzeug==2.3.7
```